src/shopDetail/_components/organizer/Organizer.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback, useMemo } from 'react'
+import { Linking, Platform } from 'react-native';
+
+import { Icon, Typography } from '../../../_components';
+import { StorageKey } from '../../../_models';
+import { storage } from '../../../storage';
+import { useGetOrganizer } from '../../_queries/useGetOrganizer';
+import * as Styled from './style';
+
+export const Organizer = ({ id }: { id: string }) => {
+  const { data, isLoading } = useGetOrganizer({ id });
+
+  const formattedAddress = useMemo(() => {
+    const lang = storage.getString(StorageKey.Language);
+    // Fallback to nl, in case there is no translated address
+    const address = data?.address[lang] || data?.address['nl'];
+    if (!address) return '';
+
+    let addressString = address.streetAddress;
+    if (addressString.trim()) addressString += ', ';
+    addressString += address.postalCode;
+    addressString += ' ';
+    addressString += address.addressLocality;
+    return addressString.trim();
+  }, [data?.address]);
+
+  // TODO: check with JM if this can happen
+  const name = useMemo(() => {
+    const lang = storage.getString(StorageKey.Language);
```

Comment:
dit zou wel nog wat kunnen terugkomen dus is misschien handiger om in util te steken